src/main/java/com/socrata/datasync/GISJob.java

```diff
@@ -0,0 +1,78 @@
+package com.socrata.datasync;
+
+import com.socrata.datasync.PublishMethod;
+
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.Serializable;
+
+/**
+ * ******************************************************
+ * ******************************************************
+ * ************ IMPORTANT NOTICE PLEASE READ ************
+ * ******************************************************
+ *
+ * The purpose of this class is solely for backwards compatability for
+ * opening old .sij files (saved from DataSync 0.2 or earlier). The
+ * "real"/current implementation of this class is:
+ * com.socrata.datasync.job.IntegrationJob
+ *
+ * ******************************************************
+ * ******************************************************
+ * ******************************************************
+ */
```

Comment:
This file probably shouldn't exist. It's unlikely DataSync 0.2 or earlier will have saved a GISJob. The other GISJob class in the jobs directory should be sufficient.